kakao/src/main/kotlin/com/agoda/kakao/Actions.kt

```diff
@@ -790,3 +791,34 @@ interface WebActions {
         web.withElement(ref).perform(action)
     }
 }
+
+/**
+ * Provides action for TabLayout
+ */
+interface SpinnerActions : BaseActions {
+    /**
+     * Selects a spinner item at given position
+     *
+     * @param position Int
+     */
+    fun selectItem(position: Int) {
+        click()
+        // Clicks on the first matched item from the list.
+        Espresso.onData(Matchers.allOf(Matchers.`is`(Matchers.instanceOf(String::class.java))))
+            .atPosition(position)
+            .perform(ViewActions.click())
+    }
+
+    /**
+     * Selects a spinner item at given string content
+     *
+     * @param content String
+     */
+    fun selectItem(content: String) {
+        click()
+        // Clicks on the first matched item with content string from the list.
+        Espresso.onData(Matchers.allOf(Matchers.`is`(Matchers.instanceOf(String::class.java)),
```

Comment:
Same here. I think you should at least specify some bounds, like adding a root matcher (if spinner creates another window), basically bound data by parent view, to protect this class from accessing non relevant data.